octobot_services/interfaces/util/portfolio.py

```diff
@@ -109,6 +109,18 @@ def get_global_portfolio_currencies_amounts():
     return real_global_portfolio, simulated_global_portfolio
 
 
+def get_global_portfolio_currencies_values():
+    currencies_values = {}
+    for exchange in interfaces.get_exchange_managers():
+        this_currency_values = (
+            exchange.exchange_personal_data.portfolio_manager.portfolio_value_holder.current_crypto_currencies_values
```

Comment:
We try to not call directly internal logic but to create an api in octobot-trading. For example here, if it doesn't already exist, we have to create an api to request current crypto values from an exchange. This prevent having to change OctoBot-Services when modifying OctoBot-Trading internals.

Reply:
Okay I'll do that